### DISCUSSION TITLE
Guess 🎥🍿 catchphrase

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1071] [Galaxy Trivia] [answered]
> 🛣️🛣️? Where we're going we don't need 🛣️🛣️.

**Bonus:** Name the character and actor.

**Comment by NomanGhiasy** [comment]
Where we're going, we don't need roads.

Dr. Emmett Brown ( Doc Brown ) is the character name played by Christopher Lloyd.

**Comment by peterbaccaro** [comment, marked as answer]
**Roads?! Where we're going We don't need Roads!!!** is a famous quote from the movie Back to the Future. The quote is spoken by the character **Dr. Emmett Brown, often referred to as Doc**, portrayed by **Christopher Lloyd**.

**Reply by NomanGhiasy** [reply]
so I was right, thanks for confirming hehe

**Comment by thuongtruong1009** [comment]
The 🎥🍿 catchphrase "🛣️🛣️? Where we're going we don't need 🛣️🛣️." is from the movie "Back to the Future."

The character who says this catchphrase is Dr. Emmett "Doc" Brown, portrayed by actor Christopher Lloyd.
